jsonpaths/com.snowplowanalytics.snowplow/r2f_data/jsonschema/1-0-0

```diff
@@ -0,0 +1,98 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for R2F (Right to be forgotten) data files that contain the identifiers that should be deleted",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "r2f_data",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "pojo": {
+      "description": "A scalar field from the EnrichedEvent POJO (e.g. user_id)",
+      "type": "object",
+      "properties": {
+        "fieldName": {
+          "enum": [
+            "user_id",
+            "user_ipaddress",
+            "user_fingerprint",
+            "domain_userid",
+            "network_userid",
+            "ip_organization",
+            "ip_domain",
+            "tr_orderid",
+            "ti_orderid",
+            "mkt_term",
+            "mkt_content",
+            "se_category",
+            "se_action",
+            "se_label",
+            "se_property",
+            "mkt_clickid",
+            "refr_domain_userid",
+            "domain_sessionid"
+          ],
+          "description": "The name of the field"
+        },
+        "valueForWhichRowWillBeDeleted": {
```

Comment:
Any chance for better name? `valueToDelete` or just `value` having that we don't call `filedName` a `fieldWhereValueForWhichRowWillBeDeletedShouldBeFound`?

Reply:
Please suggest a better name that both:

* Does state intent.
* Does not mislead the user into believing that the value alone, instead of the entire row will be deleted.